manifests/params.pp

```diff
@@ -40,7 +41,7 @@
   $ui_package_ensure     = 'latest'
   $ui_package_name       = 'consul_ui'
   $user                  = 'consul'
```

Comment:
Not sure why I didn't think of doing that in the prams vs trying to remap it in the init.pp ... lost in the moment.  But yes:  I deliberately attempted to clear user statements as they aren't needed in the container.

I'll rework it tonight/next few days.  Thanks for the input.